std_msgs/Duration duration
---
ml4kp_bridge/SpacePoint final_state